BangazonAPI/TestBangazonAPI/DatabaseFixture.cs

```diff
@@ -1,12 +1,83 @@
-﻿using System;
+﻿using BangazonAPI.Models;
+using Microsoft.Extensions.Configuration;
+using System;
 using System.Collections.Generic;
-using System.Linq;
+using System.Data.SqlClient;
 using System.Text;
-using System.Threading.Tasks;
 
 namespace TestBangazonAPI
 {
-    class DatabaseFixture
+    public class DatabaseFixture : IDisposable
     {
+        private readonly string ConnectionString = @$"Server=localhost\SQLEXPRESS01;Database=BangazonAPI;Trusted_Connection=True;";
+
+        //Creates two product objects for testing: one for the GET, POST, and PUT tests, and another for the DELETE method
+        public Product TestProduct { get; set; }
+        public Product TestDeleteProduct { get; set; }
+        public DatabaseFixture()
+        {
+            //Object for GET, POST, PUT
+            Product newProduct = new Product
+            {
+                ProductTypeId = 1,
```

Comment:
It's okay to leave these in for right now, but I do want to put on your radar that this may break down the road if someone deletes the product type with an id of 1 and/or a customer with an id of 1. Your options are (from least labor intensive to most labor intensive): 
1. Leave it until it breaks
2. Leave it until the customer and product type tickets are done, and then coordinate with the test data for those tickets so that you _know_ the product type and customer exist in the DB when the tests run
3. Insert a customer and a product type the same way you're inserting your two products

Reply:
To clarify: the test data you gave in the instructions is great, but it will only create a customer and product type with an id of 1 if the database is currently empty. And even then, these tests will stick around throughout the life cycle of the project, so there's a reasonable chance that at some point we could delete that user or that payment type and get a false negative on these tests